### PR TITLE
Kai default configuration has a typo

### PR DESCRIPTION
**Pull request by Toure**
kai.yaml contained a typo in regards to which keyword
golang was looking for to set the http.timeout-seconds mapstructure.
The following changes updates the config file and the required golden
test file to reflect this change.

Signed-off-by: Toure Dunnon <toure.dunnon@anchore.com>

fixes: https://github.com/anchore/kai/issues/33

### CODE REVIEW

**Comment by found-it** [comment]
@Toure sorry I just saw this. Looks like the unit tests for the configuration are failing

File: kai/lib.go [hunk 1]
```diff
@@ -92,25 +92,7 @@ func GetInventoryReport(cfg *config.Application) (inventory.Report, error) {
...
-				select {
-				case <-ch.stopper:
-					return
-				default:
-					fetchPodsInNamespace(clientset, cfg.Kubernetes, namespace, ch)
-				}
-			}
-		}()
-	}
+	getPodsFromNameSpace(cfg, kubeconfig, ch, queue)
```
Comment (found-it) [inline kai/lib.go:95]: Love it, good idea to break this out. One nitpick - the function name is very similar to `fetchPodsInNamespace`, what do you think about making it `launchPodWorkerPool` or something like that?

File: .gitignore [hunk 2]
```diff
@@ -31,6 +31,9 @@ coverage.txt
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+# vim swap files
+*.swp
```
Comment (found-it) [inline .gitignore:35]: What do you think about `*.sw?` since vim generates .swp, .swo, .swn, etc. for it's various swap files

File: kai.yaml [hunk 3]
```diff
@@ -54,4 +54,4 @@ anchore:
   password: $KAI_ANCHORE_PASSWORD
 #  http:
 #    insecure: true
-#    timeoutSeconds: 5
+#    timeout-seconds: 5
```
Comment (dakaneye) [inline kai.yaml:57]: Yo man, not sure why but I think I used camelCase on purpose. We might want to dig into why mapstructure isn't picking this up. If you already know can you just reply to this comment and let me know?

Comment (found-it) [reply kai.yaml:57]: Hey @dakaneye! I tested the code out and the mapstructure renaming works if I make the tag camelCase. 

```
	TimeoutSeconds int  `mapstructure:"timeoutSeconds"`
```

I think mapstructure is parsing it correctly. Is there another reason why this should be camelCase vs the other configuration options?

Comment (Toure) [reply kai.yaml]: CamelCase is the way of golang 😄

**Review by found-it — APPROVED** [review]
lgtm

**Pull request merged**